flake.nix

```diff
@@ -23,10 +23,6 @@
         let
           # Define versions of Penumbra and CometBFT
           penumbraRelease = null; # Use the local working copy
-          # penumbraRelease = { # Use a specific release
-          #   version = "0.71.0";
-          #   sha256 = "sha256-2mpyBEt44UlXm6hahJG9sHGxj6nzh7z9lnj/vLtAAzs=";
-          # };
```

Comment:
i found this previous comment of yours to be a friendly invitation to tinkering, but i don't mind removing this either! :woman_shrugging: